test/e2e/admin_test.go

```diff
@@ -0,0 +1,79 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+
+	"github.com/argoproj/gitops-engine/pkg/utils/kube"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/argoproj/argo-cd/v2/test/e2e/fixture"
+	fixtureutils "github.com/argoproj/argo-cd/v2/test/e2e/fixture/admin/utils"
+	appfixture "github.com/argoproj/argo-cd/v2/test/e2e/fixture/app"
+
+	. "github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1"
+	. "github.com/argoproj/argo-cd/v2/test/e2e/fixture/admin"
+)
+
+func TestBackupExportImport(t *testing.T) {
+	var exportRawOutput string
+	ctx := Given(t)
+	// Create application in argocd namespace
+	appctx := appfixture.GivenWithSameState(t)
+
+	// Create application in test namespace
+	appctx.
+		Path(guestbookPath).
+		Name("exported-app1").
+		When().
+		CreateApp().
+		Then().
+		And(func(app *Application) {
+			assert.Equal(t, "exported-app1", app.Name)
+			assert.Equal(t, fixture.TestNamespace(), app.Namespace)
+		})
+
+	// Create app in other namespace
+	appctx.
+		Path(guestbookPath).
+		Name("exported-app-other-namespace").
+		SetAppNamespace(fixture.AppNamespace()).
+		When().
+		CreateApp().
+		Then().
+		And(func(app *Application) {
+			assert.Equal(t, "exported-app-other-namespace", app.Name)
+			assert.Equal(t, fixture.AppNamespace(), app.Namespace)
+		})
+
+	ctx.
+		When().
+		RunExport().
+		Then().
+		AndCLIOutput(func(output string, err error) {
```

Comment:
Lets create And consequence which will parse output and return result with error if needed